pages/category/index.vue

```diff
@@ -73,13 +73,91 @@
                   class="navbar__button navbar__button--back"
                   :link="breadcrumbs[breadcrumbs.length - 2].link"
                 >
-                  <SfIcon size="18px" color="#43464E" icon="chevron_left" />Back
+                  <SfIcon size="0.813rem" color="#2B2B2B" icon="chevron_left" />Back
                 </SfLink>
               </SfList>
             </div>
           </SfLoader>
         </LazyHydrate>
       </div>
+      <SfLoader
+        :class="{ 'loading--products': productSearchLoading }"
+        :loading="productSearchLoading"
+      >
+        <div class="products" v-if="!loading">
+          <transition-group
+            v-if="isGridView"
+            appear
+            name="products__slide"
+            tag="div"
+            class="products__grid"
+          >
+            <KiboProductCard
+              v-for="(product, i) in products"
+              :key="productGetters.getSlug(product)"
```

Comment:
use product code for key, not all products will have slug

Reply:
Resolved